common/app/mvt/MultiVariateTesting.scala

```diff
@@ -45,7 +45,9 @@ case class TestDefinition (
     name,
     description,
     conf.Off,
-    sellByDate)
+    sellByDate,
+    exposeClientSide = false
```

Comment:
I think this should be true, there are at most 9 tests, and so far all the server tests have used the switch value. @uplne